server/server_test.go

```diff
@@ -1894,25 +1936,32 @@ func getStmtCnt(content string) (stmtCnt map[string]int) {
 
 const retryTime = 100
 
-func (cli *testServerClient) waitUntilServerCanConnect() {
+func (cli *testServerClient) waitUntilCustomizationServerCanConnect(dsn string) {
```

Comment:
Is there any specific reason for making `dsn` as a parameter?

Reply:
remove custom DSN and ```waitUntilCustomizationServerCanConnect```.